### PR TITLE
cephfs-shell: Fix flake8 errors (E302, E502, E128, F821, W605, 5128, E122)

### PR DESCRIPTION
**Pull request by diru1100**
cephfs-shell: Fix flake8 errors (E302, E502, E128, F821, W605, 5128, E122)

Before errors in file cephfs-shell:
first

After Fix:
last

Fixes: https://tracker.ceph.com/issues/44645
Signed-off-by: Dandamudi Rohit <rohitdandamudi.1100@gmail.com>
## Checklist
- [X] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by diru1100** [comment]
Hello @varshar16  

Can you please have a look at it. Also the W605 didn't appear when I ran flake8 command, but corrected it anyway.
 
Thanks

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -136,6 +136,7 @@ def ls(path, opts=''):
         perror(e)
         shell.exit_code = e.get_error_code()
 
+
```
Comment (varshar16) [inline src/tools/cephfs/cephfs-shell]: Move this change to a different commit.

File: src/tools/cephfs/cephfs-shell [hunk 2]
```diff
@@ -1357,7 +1357,6 @@ class CephFSShell(Cmd):
         else:
             self.perror("snapshot can only be created or deleted; check - "
                         "help snap")
-            self.exit_code = e.get_error_code()
```
Comment (varshar16) [inline src/tools/cephfs/cephfs-shell]: This change is wrong.  Exit code is required and it should return `1` here .

**Comment by varshar16** [comment]
Please fix these errors too:
```
../src/tools/cephfs/cephfs-shell:1135:48: E226 missing whitespace around arithmetic operator
../src/tools/cephfs/cephfs-shell:1140:40: E226 missing whitespace around arithmetic operator
```
